src/app/views/query-runner/request/feedback/feedbackWrapper.ts

```diff
@@ -2,6 +2,7 @@ import '@ms-ofb/officebrowserfeedbacknpm/intl/en/officebrowserfeedbackstrings';
 import { getTheme } from 'office-ui-fabric-react';
 import { geLocale } from '../../../../../../src/appLocale';
 import { translateMessage } from '../../../../utils/translate-messages';
+import * as packageJson from '../../../../../../package.json';
```

Comment:
The version number is going to be needed in two upcoming tasks. Telemetry and displaying the feature. This should be extracted to a reusable function that exports the version so that it is easy to reuse

Reply:
Okay I'll change it into a function